depends/README.md

```diff
@@ -12,15 +12,16 @@ For example:
 
     make HOST=x86_64-w64-mingw32 -j4
 
-**Bitcoin Core's configure script by default will ignore the depends output.** In
+**Bitcoin Core's `configure` script by default will ignore the depends output.** In
 order for it to pick up libraries, tools, and settings from the depends build,
-you must point it at the appropriate `--prefix` directory generated by the
-build. In the above example, a prefix dir named x86_64-w64-mingw32 will be
+you must set `CONFIG_SITE` environment variable to appropriate
```

Comment:
you must set the `CONFIG_SITE` environment variable to point to a `config.site` settings file. In the above example, a file named `depends/x86_64-w64-mingw32/share/config.site` will be created. To use it during compilation:

CONFIG_SITE=$PWD/depends/x86_64-w64-mingw32/share/config.site ./configure

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/17934#issuecomment-816556534).